### ISSUE TITLE
IllegalArgumentException at fetchImagesForDepictedItem

### ISSUE THREAD

**Issue by nicolas-raoul** [commons-app/apps-android-commons#3763]
1. Go to any item (for instance "egg" or "rabbit"), either via Explore or via a depiction in Media Details.
2. Scroll the pictures that depict that item.
3. Crash:
```
USER_COMMENT=scrolling pics in Explore item
APP_VERSION_CODE=561
APP_VERSION_NAME=2.12.3-debug-master
ANDROID_VERSION=9
PHONE_MODEL=SM-G970F
STACK_TRACE=java.lang.IllegalArgumentException: Parameter specified as non-null is null: method kotlin.jvm.internal.Intrinsics.checkParameterIsNotNull, parameter query
	at fr.free.nrw.commons.explore.depictions.DepictsClient.fetchImagesForDepictedItem(Unknown Source:3)
	at fr.free.nrw.commons.depictions.Media.DepictedImagesPresenter.fetchMoreImages(DepictedImagesPresenter.java:87)
	at fr.free.nrw.commons.depictions.Media.DepictedImagesFragment$1.onScroll(DepictedImagesFragment.java:139)
	at android.widget.AbsListView.invokeOnItemScrollListener(AbsListView.java:2200)
	at android.widget.AbsListView.trackMotionScroll(AbsListView.java:8258)
	at android.widget.AbsListView$FlingRunnable.run(AbsListView.java:7644)
	at android.view.Choreographer$CallbackRecord.run(Choreographer.java:949)
	at android.view.Choreographer.doCallbacks(Choreographer.java:761)
	at android.view.Choreographer.doFrame(Choreographer.java:693)
	at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:935)
	at android.os.Handler.handleCallback(Handler.java:873)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loop(Looper.java:214)
	at android.app.ActivityThread.main(ActivityThread.java:7032)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:494)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:965)

IS_SILENT=false
USER_EMAIL=
USER_CRASH_DATE=2020-05-21T21:43:45.179+09:00
REPORT_ID=0da939d9-75bc-43ff-833f-c05e0e780fdc
```
If the item has few depictions (for instance "dishwasher"), the crash is immediate, no need to scroll.

**Comment by macgills** [comment]
There is also a problem with child/parent tabs but that is resolved in #3468 as I went through the adapters and fixing some bugs to get them rendering

**Comment by nicolas-raoul** [comment]
Thanks, it works now :-)
